### ISSUE TITLE
Does sctransform normalisation account for changes in RNA composition?

### ISSUE THREAD

**Issue by lc822** [ChristophH/sctransform#83]
Hi,

I was wondering whether the sctransform normalisation is able to account for changes in RNA composition between conditions.  I have a dataset of non-activated and TCR-activated T cells.  I see some genes with reduced expression upon stimulation and I am wondering whether some of these genes could be artificially decreased due to changes in RNA composition.  If activation-associated genes increase in expression and take up a larger fraction of the RNA pool following stimulation, this could lead to undersampling of the RNAs that don't change in expression.  Does this make sense and does sctransform have any way to deal with this e.g. something like the TMM method in EdgeR?

Many thanks,
Lucy

**Comment by ChristophH** [comment]
No, sctransform does not account for changes in the RNA composition. However, I don't think compositional effects are that strong between non-activated and TCR-activated T cells. Compositional bias is more likely a problem when the diversity, the effective number of genes, is low (exp(H) where H is the Shannon entropy). I assume that your effective number of genes is in the hundreds and relatively constant across the conditions that you compare. (Note that sequencing depth has an influence as well)
